insights/v1alpha1/types_insights.go

```diff
@@ -171,7 +171,7 @@ type GathererStatus struct {
 	// lastGatherDuration represents the time spent gathering.
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:Type=string
-	// +kubebuilder:validation:Pattern="^([1-9][0-9]*(\\.[0-9]+)?(ns|us|µs|ms|s|m|h))+$"
+	// +kubebuilder:validation:Pattern="^(([0-9]+(?:\\.[0-9]+)?(ns|us|µs|μs|ms|s|m|h))+)$"
```

Comment:
Sorry I don't understand why. `0m59s` is valid Go duration so why wuold you limit it?

Reply:
We don't presently allow it, and the bug is because of secondary units not allowing zeros right? 

Just a thought, doesn't necessarily need to be fixed, but if it were easy I'd have preferred not to allow leading zeros

Reply:
OK I see. Honestly I am not that proficient in regular expressions. I have experimented a little bit and perhaps something like the following could work:
```
^(([1-9]\d*(\.\d+)?(h|m|s|ms|ns|us|µs))|(0(\.\d+)?(s|ms|ns|us|µs)))+$
```
The tricky thing is for example `0.2m` which is also a valid Go duration. So I think I would lean more towards the existing expression.

Reply:
Ack, lets keep as it is then, thanks for checking